offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -235,8 +306,10 @@ class QuotationDetails {
      * @param productPropertySpacing A spacing factor assigned for that text
      * @return An estimation of the space needed for the given text
      */
-    private static int calculateItemSpace(String productProperty, ProductPropertySpacing productPropertySpacing) {
-        return Math.ceil(productProperty.length() / productPropertySpacing.getCharsLineLimit())
+    private static double calculateItemSpace(String productProperty, ProductPropertySpacing productPropertySpacing) {
+
+        // As soon as the PropertySpacing limit is reached a line break will occur
+        return Math.ceil(productProperty.length() / productPropertySpacing.getCharsLineLimit()) * lineSpace
```

Comment:
what line space do we have here? I am a little confused with all the line spaces

Reply:
Since this is hard to explain i added a picture showing the difference between the linespaces 
![example](https://user-images.githubusercontent.com/29627977/129552905-47b2deb6-1bf1-410a-b246-da8f2b245ad5.png)

Reply:
The starting point of an item is always one line with a bigger font listing the item attributes and a whitespace line underneath. This is slightly bigger than following linespaces in the description itself.
In theory the best way would be to calculate the linespace for the bigger and smaller font seperately but this is more hassle than it's worth since only the description adjusts the height necessary for an item in 99% of the cases

Reply:
How about then calling the minimalLineSpace minimalItemLineSpace? The other can be left as is.

Reply:
Thanks for explaining

Reply:
Good Idea I'll add that! 🧠